packages/core/src/data-module/types.ts

```diff
@@ -104,6 +104,7 @@ export interface DataStream<T extends Primitive = Primitive>
   associatedStreams?: StreamAssociation[];
   isLoading?: boolean;
   isRefreshing?: boolean;
+  assetName?: string;
```

Comment:
I don't think this is the correct way to get the asset name into the data stream. DataStream is a generic concept separate from SiteWise. For example, the TwinMaker data source also uses this.

I'm going to talk with my team and recommend a different solution.

Reply:
Ok, Thank you.